app/views/shared/_footer.html.erb

```diff
@@ -27,7 +27,8 @@
 
   <div class="footerExtras">
     <div>
-      <%= select_tag 'locale', options_for_select(@locales.collect{ |l| [l[:name], l[:locale]] }, @locale), 'aria-label': t('language') %>
+      <%= label_tag 'locale', "Choose a Language", class: 'visually-hidden' %>
+      <%= select_tag 'locale', options_for_select(@locales.collect{ |l| [l[:name], l[:locale]] }, @locale) %>
```

Comment:
while we're making changes here can we change the iterator from `l` to `locale` or something similar for improved readability

Reply:
That is a great idea. I'll do that while I'm in the code.